ihatemoney/tests/common/ihatemoney_testcase.py

```diff
@@ -1,45 +1,21 @@
 import os
-from unittest.mock import MagicMock
+import unittest
 
-from flask_testing import TestCase
+import pytest
 from werkzeug.security import generate_password_hash
 
 from ihatemoney import models
-from ihatemoney.currency_convertor import CurrencyConverter
-from ihatemoney.run import create_app, db
 
 
-class BaseTestCase(TestCase):
+@pytest.mark.usefixtures("client", "converter")
+class BaseTestCase(unittest.TestCase):
```

Comment:
Inherinting from `unittest.TestCase` is not required with pytest.

https://docs.pytest.org/en/7.1.x/getting-started.html#group-multiple-tests-in-a-class